x-pack/metricbeat/module/istio/_meta/config.reference.yml

```diff
@@ -32,3 +32,10 @@
   period: 10s
   # use istio-pilot.istio-system:15014, when deploying Metricbeat in a kubernetes cluster as Pod or Daemonset
   hosts: ["localhost:15014"]
+
+# Istio istiod to monitor the Istio Daemon for versions after 1.5 of Istio.
+- module: istio
+  metricsets: ['istiod']
+  period: 10s
+  # use istiod.istio-system:15014, when deploying Metricbeat in a kubernetes cluster as Pod or Daemonset
+  hosts: ['localhost:15014']
```

Comment:
Is there a reason to use different config blocks for metricsets that use the same endpoint and period?

Reply:
Actually the endpoint would be different for each different metricset in a real setup. Different metricsets for different k8s services. It will be `localhost` only when there is something like port-forwarding which is not the normal case but it is useful for testing scenarios.